### PR TITLE
extend markup of openvpn-openssl-exception to accomodate psycopg2 ope…

### PR DESCRIPTION
**Pull request by xsuchy**
…nssl exception

Fixes: #2038

### CODE REVIEW

**Comment by jlovejoy** [comment]
@swinslow - this looks good to me, agree?

**Review by swinslow — CHANGES_REQUESTED** [review]
One question as I'm not sure whether this picks up the exact match from #2038 -- please see my comment inline.

File: src/exceptions/openvpn-openssl-exception.xml [hunk 1]
```diff
@@ -18,7 +19,8 @@
          the same license as OpenSSL), and distribute linked
          combinations including the two. You must obey the GNU <optional>Lesser</optional> General
          Public License in all respects for all of the code used other
-         than OpenSSL. If you modify this file, you may extend this
+         than OpenSSL. If you modify this <alt name="file" match="file(\(s\))?">file</alt><alt name="this exception" match="()|( with this exception)"></alt>,
```
[the remark this answers is not in the document]
Comment (xsuchy) [reply src/exceptions/openvpn-openssl-exception.xml]: Good catch. Updated.

**Review by swinslow — APPROVED** [review]
LGTM, thanks @xsuchy!

**Pull request merged**